README.md

```diff
@@ -6,14 +6,18 @@ The CLI is build on the Logentries REST APIs and provides a tool to interact dir
 **Installation**
 ----------------
 
+`pip install logentries-lecli`
+
+or
+
 `pip install <url to repository>`
 
 or 
 
 `pip install <path to project directory>`
 
 **Note** 
-Lecli will look for the path of your config.ini file depending on your operating system:
+A dummy config file will be loaded to the config directory during installation. You should edit this file with your configuration after the installation. Lecli will decide the configuration file path depending on your operating system:
```

Comment:
`A dummy config file will be created in config directory during installation`

`You should edit this file with your API keys after the installation is complete`

`Depending on your operating system the Lecli config can be located at the following paths:`